baselines/dom.generated.d.ts

```diff
@@ -4722,12 +4722,12 @@ interface FileReaderEventMap {
 
 interface FileReader extends EventTarget {
     readonly error: DOMException | null;
-    onabort: ((this: FileReader, ev: ProgressEvent) => any) | null;
-    onerror: ((this: FileReader, ev: ProgressEvent) => any) | null;
-    onload: ((this: FileReader, ev: ProgressEvent) => any) | null;
-    onloadend: ((this: FileReader, ev: ProgressEvent) => any) | null;
-    onloadstart: ((this: FileReader, ev: ProgressEvent) => any) | null;
-    onprogress: ((this: FileReader, ev: ProgressEvent) => any) | null;
+    onabort: ((this: FileReader, ev: ProgressEvent & {target: FileReader}) => any) | null;
```

Comment:
i would create a new type `FilreReaderProgressEvent` that extends `ProgressEvent` and use that instead.

Reply:
Thank you for reviewing. I have added `FileReaderProgressEvent`.